.github/workflows/packaging.yaml

```diff
@@ -0,0 +1,216 @@
+name: Package linux binaries
+
+env:
+  MY_LIST: "0.10.7,0.13.6,0.14.1,0.16.1,0.17.3,0.18.3,0.19.4,0.21.1,0.24.0,0.26.5,0.28.0,0.29.1,0.31.2,0.32.2"
+
+on:
+  workflow_dispatch:
+
+jobs:
+  package-binaries:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-22.04]
+        arch: [amd64]
+
+    permissions:
+      contents: write
+      packages: write
+      id-token: write
+
+    steps:
+
+      - name: Configure AWS credentials
```

Comment:
```
      - name: Configure AWS credentials
        uses: aws-actions/configure-aws-credentials@v1-node16
        with:
          aws-access-key-id: ${{ secrets.EKS_ACCESS_KEY_ID }}
          aws-secret-access-key: ${{ secrets.EKS_SECRET_ACCESS_KEY }}
          aws-region: us-east-2
```

Reply:
We will authenticate to AWS.
The provider we use is compatible with AWS syntax, that is why the authentication is a bit different. 
we create the necessary files under ~/.aws/